webapp/components/create_post.jsx

```diff
@@ -172,7 +172,10 @@ export default class CreatePost extends React.Component {
                         const state = {};
                         state.serverError = err.message;
                         state.submitting = false;
-                        this.setState({state});
+                        this.setState({
```

Comment:
Do you mind fixing this `setState` block? Your stuff will work fine, but it doesn't actually show the error message because it's setting `this.state.state`. This block should be
```
this.setState({
    serverError: err.message,
    submitting: false,
    message: post.message
});
```